GibbsSampling.Application.Tests/Fixtures/Fixture.fs

```diff
@@ -0,0 +1,91 @@
+﻿module Fixture
+
+open BioFSharp
+
+open CompositeVector.Functions
+open PositionMatrix.Functions
+
+type Fixture() =
+    
+    member this.Pseudocount with get () = 1.
+    
```

Comment:
I think in f# this syntax can improved by using:

- a module with static parameters
- a static class (`type xxx =` without the `()`) with static members

```fsharp
type Fixture =
    static member Pseudocount = 1.
```

This shows clear intent, that these are read only values without initializing a mutable class first.

Reply:
In addition it improves access by changing `Fixture().Pseudocount` to `Fixture.Pseudocount`. No need to init a class on every parameter access.

Reply:
fixed

Reply:
still `type Fixture() =` 😅

Reply:
but with static members access seems the same, so you can leave it or update it as you want